libcxl.c

```diff
@@ -482,6 +487,12 @@ static int open_afu_dev(struct cxl_afu_h *afu, char *path)
 		errno = EPROTO;
 		goto err_close;
 	}
+
+	 /* try opening the afu descriptor attr. Dont fail if we cant. */
+	if (cxl_open_afudesc(afu)) {
+		pr_devel("Unable to open afu_desc for %s. Wont support mmio reads with all FFFFs"
```

Comment:
Thanks for the review Andrew. I have updated the pull request with the needed changes.